python/lsst/afw/geom/endpoint.cc

```diff
@@ -82,12 +83,34 @@ template <typename PyClass>
 void addMakeFrame(PyClass& cls) {
     using Class = typename PyClass::type;  // C++ class associated with pybind11 wrapper class
     // return a deep copy so Python cannot modify the internal state
-    cls.def("makeFrame", [](Class const & self) {
+    cls.def("makeFrame", [](Class const& self) {
         auto frame = self.makeFrame();
         return frame->copy();
     });
 }
 
+// Endpoint comparison useful in Python but not C++
```

Comment:
I'm not convinced by this assertion. If it would not be a lot more work I recommend making this available to C++.